circuits/circom/test/ecdh_test.circom

```diff
@@ -1,3 +1,4 @@
-include "../ecdh.circom"
+pragma circom 2.0.0;
+include "../ecdh.circom";
 
-component main = Ecdh();
+component main {public [pubKey]} = Ecdh();
```

Comment:
Could you elaborate a little bit on this syntax?

Reply:
Sure! In circom 2, all input signals are private by default. The only way to declare a signal as public is in its `component main ...` instantiation.